src/content/app-platform/defaulting-validation/index.md

```diff
@@ -120,14 +120,14 @@ If app-operator finds a matching [AppCatalogEntry]({{< relref "/ui-api/managemen
 - Cloud provider compatibility (e.g. you can’t install the azure-ad-pod-identity app in AWS).
 - Namespace restriction (cluster singleton, namespace singleton, fixed namespace).
 
-## GitOps support for Flux
+## GitOps support
 
 If you are managing your App CRs with Flux or a similar GitOps tool then the
 validating webhook may block creation of the App CR if it is created before the
 referenced configmap or secret.
 
-To prevent this you can add the label `giantswarm.io/managed-by` set to `flux`.
-e.g.
+To prevent this you can add the label `giantswarm.io/managed-by` set the value
+to the tool you're using e.g. `flux`.
```

Comment:
We changed this recently in https://github.com/giantswarm/giantswarm/issues/20728 
To widen the scope so any value for the label is accepted.